ivy.xml

```diff
@@ -49,6 +49,8 @@
     <dependency org="log4j" name="log4j" rev="1.2.16" transitive="false" conf="default"/>
     <dependency org="jline" name="jline" rev="0.9.94" transitive="false" conf="default"/>
 
+    <dependency org="com.google.code.findbugs" name="annotations" rev="3.0.1" conf="default"/>
```

Comment:
Any alternative other than intro dependency to google libraries. Presently, zk code doesn't have any dependency with google libraries.